src/main/java/edu/RobertoAranovich/advancedjava/Author.java

```diff
@@ -0,0 +1,74 @@
+package edu.RobertoAranovich.advancedjava;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * A simple Author Class to store a book's author info
+ *
+ * @author Roberto Aranovich
+ */
+
+public class Author {
+
+    private String name;
+    private Date dateOfBirth;
+    private Date dateOfDeath;
+    private String placeOfBirth;
+
+    /**
+     * Creates a new Author instance.
+     *
+     * @param name         author's first and last name (and middle initial if any).
+     * @param dateOfBirth  author's date of birth.
+     * @param dateOfDeath  author's date of death ('00-00-0000' if alive).
+     * @param placeOfBirth author's place of birth
+     */
+    public Author(String name, Date dateOfBirth, Date dateOfDeath, String placeOfBirth) {
```

Comment:
a better example of DI would be have used interfaces here rather than concrete classes.

Reply:
I'm not quite sure about how to use interfaces for dependency injection. Any suggestions?